docs/document/docusaurus.config.js

```diff
@@ -0,0 +1,163 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/okaidia');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const versions = require('./versions.json');
+
+function getLatestVersionName() {
+  return versions[0];
+}
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'kintone UI Component · Be a smart kintone developer.',
+  tagline: 'It will support your UI customization project.',
+  url: 'https://kintone-ui-component.netlify.app',
+  baseUrl: '/',
+  onBrokenLinks: 'warn', // throw
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/kuc_box_yellow.png',
+
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: 'kintone-ui-component', // Usually your GitHub org/user name.
+  projectName: 'kintone-labs', // Usually your repo name.
+
+  // Even if you don't use internalization, you can use this field to set useful
+  // metadata like html lang. For example, if your site is Chinese, you may want
+  // to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: 'en',
```

Comment:
About the English page path, it will be changed to not needed to add "en/".
I'm concerned about the redirect setting from the old URL that includes "en/" and updating cost of all related document (github, README, github discussions, some user-side article, etc). Am I worrying too much and no need to care about it?

Reply:
In the next few months, we will don't use Netlify and the domain https://kintone-ui-component.netlify.app/ won't be accessible. the new domain is ui-component.kintone.dev we have to update all related document. So I think we don;t need to care about 'en/' on url

Reply:
Thank you very much for the implementation.
Where is the script of that?

After changing to ui-component.kintone.dev, the redirect setting is also needed. In that case, it will need the same kind of function as same as Netlify.
Can we do that?

And the final confirmation, we cannot include "/en" path to the new docusaurus site, right?

Reply:
I got it. I wanna confirm it in the MTG.
Let's note this as an issue and investigate it more later on.

Reply:
Discussion memo:
You added a script to redirect from the URL including /en/ path to the URL not including /en/. So users can also access with the old URL.

After hosting on AWS Amplify and changing to the custom domain, we should redirect from the old URL to the new URL.
  A. While providing Netlify site, we can set the redirect setting from the old URL to the new URL. (We need to develop one line JS script on Netlify site)
  B. After we end providing Netlify site, we cannot set the redirect setting from the old URL to the new URL, so we need to announce to users about that.

We should set transferring terms to let users change the URL from the old to the new.